core/lib/spree/app_configuration.rb

```diff
@@ -204,6 +204,11 @@ class AppConfiguration < Preferences::Configuration
     #   @return [Boolean] Request instructions/info for shipping (default: +false+)
     preference :shipping_instructions, :boolean, default: false
 
+    # @!attribute [rw] show_deprecated_attributes
+    #   @return [Boolean] Show deprecated model attributes in HTML views and
+    #   API responses. (default: +true+)
+    preference :show_deprecated_attributes, :boolean, default: true
```

Comment:
What about `show_address_deprecated_attributes` or `show_deprecated_naming_attributes` instead? I feel like this is too generic.

Reply:
Yes I wasn't sure about this, I'd go with  `show_address_deprecated_attributes`